src/pages/index.md

```diff
@@ -1,10 +1,52 @@
 ---
-title: Overview - Adobe Commerce Cloud Tools
+title: Commerce Cloud Tools
 description: This is the overview page of the Commerce Cloud Tools.
 ---
 
-# Adobe Commerce Cloud Tools
+<Hero slots="image, heading, text"/>
 
-## Contributing
+![Commerce Cloud Docker](_images/banner-hex-violet.png)
 
-We encourage you to participate in our open documentation initiative, if you have suggestions, corrections, additions or deletions for this documentation, check out the source from [this github repo](https://github.com/adobe/gatsby-theme-spectrum-example), and submit a pull request with your contribution.
+# Commerce Cloud Tools
+
+This is the overview page of the Commerce Cloud Tools.
+
+<Resources slots="heading, links"/>
+
+#### Resources
+
+- [Adobe Commerce Developer Documentation](https://developer.adobe.com/commerce/docs) (Adobe Developer site)
+- [Adobe Commerce Documentation](https://experienceleague.adobe.com/docs/commerce.html) (Adobe Experience League site)
+
+## Overview
+
+This documentation provides resources for deploying Adobe Commerce to a Docker environment for development, testing, and automation tasks.
+
+## Discover
+
+<DiscoverBlock width="100%" slots="heading, link, text"/>
+
+### Cloud Infrastructure
```

Comment:
I recommend removing this section since it links out to ExL and the lead-in on line 23 emphasizes that this site provides info about cloud tools specifically.